tools/docker/Dockerfile.monorepo

```diff
@@ -41,35 +44,41 @@ RUN --mount=type=cache,target=/var/lib/apt/lists/,sharing=private \
         jq \
         "php${PHP_VERSION}" \
         "php${PHP_VERSION}-ast" \
+        "php${PHP_VERSION}-bcmath" \
         "php${PHP_VERSION}-cli" \
         "php${PHP_VERSION}-curl" \
         "php${PHP_VERSION}-dom" \
+        "php${PHP_VERSION}-gd" \
+        "php${PHP_VERSION}-igbinary" \
+        "php${PHP_VERSION}-imagick" \
+        "php${PHP_VERSION}-intl" \
         "php${PHP_VERSION}-mbstring" \
+        "php${PHP_VERSION}-mysqli" \
+        "php${PHP_VERSION}-pcov" \
         "php${PHP_VERSION}-xml" \
         "php${PHP_VERSION}-zip" \
+        imagemagick \
+        libssl-dev \
+        libwebp-dev \
+        libavif-dev \
```

Comment:
I'm curious as to why these are needed.

Reply:
On the one-wordpress PR, I thought I needed them to get the tests to run in the dev container, but that might have been wrong. I'm running through all of the tests without them and will push a commit to remove them if they're passing.